3-enrich/emr-etl-runner/.gitignore

```diff
@@ -21,3 +21,5 @@ deploy/*
 
 # Added as per http://ryan.mcgeary.org/2011/02/09/vendor-everything-still-applies/
 vendor/bundle/ruby
+
+.idea/
```

Comment:
We're avoiding to add environment-specific entries to gitignore.

Reply:
Will add it to my global and remove from here - you are right!